bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -508,6 +528,42 @@
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
 
+	<channel-type id="armed">
+		<item-type>Switch</item-type>
+		<label>@text/channelAlarmArmedLabel</label>
+		<description>@text/channelAlarmArmedDescription</description>
+		<category>Alarm</category>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="alarmtargetstate">
+		<item-type>String</item-type>
+		<label>@text/channelAlarmTargetStateLabel</label>
+		<description>@text/channelAlarmTargetStateDescription</description>
+		<category>Switch</category>
+		<command>
+			<options>
+				<option value="DISARMED">@text/channelOptionAlarmStateDisarmed</option>
+				<option value="ARMED">@text/channelOptionAlarmStateArmed</option>
+			</options>
+		</command>
+	</channel-type>
```

Comment:
I have a doubt, shouldn't it be rather a Switch ?

Reply:
That would be the control switch that is already there. In that case I wouldn't need this channel. This channel was explicitly added for compatibility with both Homekit and Google Assistent. I believe it was Google Assistent that required this to be a String. Things may have changed since, as I have seen there has been quite a bit of change on Google Assitent side.
See discussion here: https://community.openhab.org/t/niko-home-control-ii/44586/453?u=mherwege

Reply:
I requested confirmation: https://community.openhab.org/t/niko-home-control-ii/44586/468?u=mherwege

Reply:
Rather than "duplicating" channel, are you sure that the user could not simply create a String item linked to the Switch channel and set command options ?

That is a little strange if now Homekit and Google Assistent adds constraints to the design of a binding. especially when this is something not documented at my knowledge.

Reply:
For the Homekit security system, the required item types are Number or String (https://next.openhab.org/addons/integrations/homekit/). For Google Assistant, the controlling item needs to be a Switch (https://next.openhab.org/docs/ecosystem/google-assistant/#securitysystem-as-switch). I use neither of these, so cannot test if connecting a String item to a switch channel with mapping options would work. I would prefer being able to limit the number of channels and to me a Swich channel would indeed make most sense.

Reply:
For the Homekit security system, the required item types are Number or String (https://next.openhab.org/addons/integrations/homekit/). For Google Assistant, the controlling item needs to be a Switch (https://next.openhab.org/docs/ecosystem/google-assistant/#securitysystem-as-switch). I use neither of these, so cannot test if connecting a String item to a switch channel with mapping options would work. I would prefer being able to limit the number of channels and to me a Swich channel would indeed make most sense.

Reply:
> Rather than "duplicating" channel, are you sure that the user could not simply create a String item linked to the Switch channel and set command options ?

@lolodomo I now tried this out. Unless I missed something, it doesn't work. You cannot change type through a profile and a Switch channel does not work with a String item. Even if a profile would be available, it is only one way, from channel to item. So the only way to achieve it with one channel is by using a rule that makes the translation.